src/Cake.DotNetTool.Module/DotNetToolPackageInstaller.cs

```diff
@@ -196,7 +204,7 @@ private List<DotNetToolPackage> GetInstalledTools(string toolLocation)
             var installedTools = isInstalledProcess.GetStandardOutput().ToList();
             var installedToolNames = new List<DotNetToolPackage>();
 
-            const string pattern = @"(?<packageName>[^\s]+)\s+(?<packageVersion>[^\s]+)\s+(?<packageShortCode>[^`s])";
+            const string pattern = @"(?<packageName>[^\s]+)\s+(?<packageVersion>[^\s]+)\s+(?<packageShortCode>[^\s]+)(?:\s+(?<packageManifest>[^\s]+))?";

```

Comment:
Optionally match the package manifest, when available, for local tools.

This regex had a bug; `` `s `` should be `\s` for non-whitespace match and it also didn't match more than one character.